docs/source/sms.rst

```diff
@@ -0,0 +1,115 @@
+ingestion and a "FILEID," which is the combination of the SMS ID and Version ID and is used as a reference in SMSTable.
+
+SMSTable contains the data of the SMS reports.
+Each record is identified uniquely based on an exposure key that is generated out of a handful of columns from the
+report itself that identifies an exposure (the ROOTNAME cannot be used as this may actually change across SMS versions).
+These records are linked to a particular SMS report in the SMSFileStats table.
+
+SMSFileStats and SMSTable are both ``peewee.Model`` objects that can be used to query SMS data from the database.
+
+For example:
+
+.. code-block:: python
+
+    import pandas as pd
+    from cosmo.sms import SMSFileStats, SMSTable
+
+    # select all sms records
+    query = SMSFileStats.select()  # Returns a query object
+
+     # Put the data in a pandas DataFrame
+    df = pd.DataFrame(list(query.dicts()))
+
+    # Show data records from a single SMS report using the SMSID
+    report = SMSFileStats.get(SMSFileStats.SMSID == '123456')
+
+    # Or
+    report = SMSFileStats.get('123456')  # The SMSID is the primary key for SMSFileStats, so this syntax also works
+
+    # Or
+    report = SMSFileStats['123456']
+
+    report.exposures  # returns a peewee.Query object
+
+    list(report.exposures.dicts())  # returns a list of dictionaries of the table data
+
+    # Query the SMS Data
+    all_data = SMSTable.select()
+
+    # Query and filter SMS Data
+    fuv_data = SMSTable.select().where(SMSTable.DETECTOR == 'FUV')
+
+SMS Ingestion Rules
+-------------------
+SMS reports are ingested into the local SMS database, ``sms.db`` using a set of rules to ensure that the correct
+information is stored.
+
+Ingestion rules are as follows:
+ - If multiple versions of the same SMS is available, the most recent version is used (largest alpha-numeric Version ID).
+ - If a new version of an SMS is found with a larger Version ID, that new report will supersede the current entry for
+   that particular SMS.
+ - If data (identified by the exposure key) from one SMS is also found in another SMS, the data will be ingested from
+   the SMS report with the larger SMS ID (and therefore more recent), and will replace any existing matching records.
+
+.. note::
+
+    These rules ensure that the data in SMSTable is accurate, but can also result in "stale" reports in SMSFileStats if
+    data occurs in a more recent SMS report.
+
+    It's also possible to have SMSTable records that don't actually correspond to any COS dataset.
+    This is due to the fact that the SMS reports reflect the observation schedule, which can change a number of times
+    right up to the planned observation date.
+    As this happens, ROOTNAMEs and other exposure information is updated accordingly, and since the ROOTNAME is based on
+    an incrementing naming system, exposures can be assigned several ROOTNAMEs until the "final" schedule is established.
```

Comment:
Could also note that if the exposures don't execute for some reason (e.g. instrument goes into safe mode), those exposures will still appear in SMS.